packages/go/openapi/src/paths/asset-isolation.asset-group-tags.certify.yaml

```
@@ -0,0 +1,65 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+
+post:
+  operationId: CertifyOrDecertifyObjects
+  summary: Certify or Decertify Objects
+  description: Manually certify/decertify objects as belonging to the Zone they're aligned to
+  tags:
+    - Asset Isolation
+    - Enterprise
+  requestBody:
+    description: The request body for certifying or decertifying objects. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)